apps/condo/domains/acquiring/schema/Payment.js

```diff
@@ -121,6 +121,17 @@ const Payment = new GQLListSchema('Payment', {
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
+            if (dv === 1) {
+                // NOTE: version 1 specific translations. Don't optimize this logic
+            } else {
+                return addValidationError(`${DV_UNKNOWN_VERSION_ERROR}dv] Unknown \`dv\``)
+            }
```

Comment:
maybe move this piece of code away to some global utils? I see that we are using it more than 2 times

Reply:
Seems like this logic is model-dependent, since if we'll change dv later on new model....we should write translations for each of them. 

But some default check just for dv === 1 can be good idea.....but I think it's not goal of current ticket